Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/ContactDiaryStorageModule.kt

```diff
@@ -0,0 +1,40 @@
+package de.rki.coronawarnapp.contactdiary.storage
+
+import dagger.Module
+import dagger.Provides
+import de.rki.coronawarnapp.contactdiary.storage.dao.ContactDiaryLocationDao
+import de.rki.coronawarnapp.contactdiary.storage.dao.ContactDiaryLocationVisitDao
+import de.rki.coronawarnapp.contactdiary.storage.dao.ContactDiaryPersonDao
+import de.rki.coronawarnapp.contactdiary.storage.dao.ContactDiaryPersonEncounterDao
+import de.rki.coronawarnapp.contactdiary.storage.repo.ContactDiaryRepository
+import de.rki.coronawarnapp.contactdiary.storage.repo.DefaultContactDiaryRepository
+import javax.inject.Singleton
+
+@Module
+class ContactDiaryStorageModule {
```

Comment:
Could cause the main thread to hang if this causes the whole DB to be instantiated on DI graph instantiation.

Reply:
What is your suggestion? I haven't noticed a problem.

Reply:
I'm already addressing this...

Reply:
done.